// react-game-kit/lib/native.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.World = exports.TileMap = exports.Stage = exports.Sprite = exports.Loop = exports.Body = undefined;

var _body = require('./native-demo/src/components/body.js');

var _body2 = _interopRequireDefault(_body);

var _loop = require('./native-demo/src/components/loop.js');

var _loop2 = _interopRequireDefault(_loop);

var _sprite = require('./native-demo/src/components/sprite.js');

var _sprite2 = _interopRequireDefault(_sprite);

var _stage = require('./native-demo/src/components/stage.js');

var _stage2 = _interopRequireDefault(_stage);

var _tileMap = require('./native-demo/src/components/tile-map.js');

var _tileMap2 = _interopRequireDefault(_tileMap);

var _world = require('./native-demo/src/components/world.js');

var _world2 = _interopRequireDefault(_world);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.Body = _body2.default;
exports.Loop = _loop2.default;
exports.Sprite = _sprite2.default;
exports.Stage = _stage2.default;
exports.TileMap = _tileMap2.default;
exports.World = _world2.default;
;

(function () {
  if (typeof __REACT_HOT_LOADER__ === 'undefined') {
    return;
  }
})();

;
